### DISCUSSION TITLE
How to restore multiple file contents that have been replaced by EDIT->Relace(ctrl+H)

### DISCUSSION THREAD

**Discussion by rfrkk** [apache/netbeans#6855] [Q&A]
### question:
I found that the contents of many files replaced by Relace seemed to be unable to be restored instantly, such as the function of pressing a button to restore, or the Ctrl+Z method.

### Provide pictures:
![Snipaste_2023-12-16_06-53-36](https://github.com/apache/netbeans/assets/137764/d175daf3-8e29-4e8c-910a-2d6533cbc4e4)
![Snipaste_2023-12-16_07-02-37](https://github.com/apache/netbeans/assets/137764/fbaf46f9-f4d0-4d66-947c-05c7b004c452)

### Please give me the solution:
How should I restore the original content if I accidentally replaced it by mistake?

**Comment by mbien** [comment]
> How should I restore the original content if I accidentally replaced it by mistake?

restoring the modifications via git is likely the quickest way.
